packages/he/cspell-ext.json

```diff
@@ -0,0 +1,42 @@
+// cSpell Settings
+{
+    "id": "he",
+    "name": "Hebrew Dictionary",
+    "description": "Hebrew Dictionary",
+    // List of dictionary files to add to the global list of dictionaries
+    "dictionaryDefinitions": [
+        {
+            "name": "he",
+            "file": "./he.txt.gz",
+            "description": "Hebrew Dictionary"
+        }
+    ],
+    // Dictionaries to always be used.
+    // Generally left empty
+    "dictionaries": [],
+    // Language Rules to apply to matching files.
+    // Files are matched on `languageId` and `local`
+    "languageSettings": [
+        {
+            // VSCode languageId. i.e. typescript, java, go, cpp, javascript, markdown, latex
+            // * will match against any file type.
+            "languageId": "*",
+            // Language local. i.e. en-US, de-AT, or ru. * will match all locals.
+            // Multiple locals can be specified like: "en, en-US" to match both English and English US.
+            "local": "*",
```

Comment:
Local needs to be set to "he", otherwise the Hebrew dictionary will always be used and cannot be turned off easily.
```
"local": "he",
```

Reply:
Done